cluster/gce/manifests/kube-apiserver.manifest

```diff
@@ -45,6 +45,8 @@
         "port": 8080,
         "path": "/healthz"
       },
+      "periodSeconds": 3,
+      "failureThreshold": 1,
```

Comment:
I would rather leave the failureThreshold in line with the liveness probes. If we want to fail faster lets decrease the period to 1 second. We should get a more reliable signal and generally fail just as quickly.

Reply:
Sounds good. Thanks for taking the time to think this over with me.